python/cuml/benchmark/bench_helper_funcs.py

```diff
@@ -91,6 +93,60 @@ def _build_fil_classifier(m, data, arg={}, tmpdir=None):
                   storage_type=arg["storage_type"])
 
 
+def _build_fil_skl_classifier(m, data, arg={}, tmpdir=None):
```

Comment:
what does it mean if tmpdir is the default of None? Probably fails? Do you want to use the temporary file module to pick a default tmpdir then?

Reply:
The default values are not necessary, removed them.